doc/modules/classification_threshold.rst

```diff
@@ -0,0 +1,171 @@
+Characteristic (ROC) statistics and 2 that uses the Precision-Recall statistics.
+
+- `"max_tpr_at_tnr_constraint"`: maximizes the True Positive Rate (TPR) such that the
+  True Negative Rate (TNR) is the closest to a given value.
+- `"max_tnr_at_tpr_constraint"`: maximizes the TNR such that the TPR is the closest to
+  a given value.
+- `"max_precision_at_recall_constraint"`: maximizes the precision such that the recall
+  is the closest to a given value.
+- `"max_recall_at_precision_constraint"`: maximizes the recall such that the precision
+  is the closest to a given value.
+
+For these options, the `constraint_value` parameter needs to be defined. In addition,
+you can use the `pos_label` parameter to indicate the label of the class of interest.
+
+The final strategy maximizes a custom utility function. This problem is also known as
+cost-sensitive learning. The utility function is defined by providing a dictionary
+containing the cost-gain associated with the entries of the confusion matrix. The keys
+are defined as `{"tn", "fp", "fn", "tp"}`. The class of interest is defined using the
+`pos_label` parameter. Refer to :ref:`cost_sensitive_learning_example` for an example
+depicting the use of such a utility function.
+
+Important notes regarding the internal cross-validation
+-------------------------------------------------------
+
+By default :class:`~sklearn.model_selection.TunedThresholdClassifier` uses a
+5-fold stratified cross-validation to tune the cut-off point. The parameter
+`cv` allows to control the cross-validation strategy. It is possible to go
+around cross-validation by passing `cv="prefit"` and provide an already fitted
+classifier. In this case, the cut-off point is tuned on the data provided to
+the `fit` method.
+
+However, you should be extremely careful when using this option. You should never use
+the same data for training the classifier and tuning the cut-off point at the risk of
+overfitting. Refer to :ref:`tunedthresholdclassifier_no_cv` that shows such overfitting. If
+you are in a situation where you have limited resources, you should consider using
+a float number that will use a single split internally.
+
+The option `cv="prefit"` should only be used when the provided classifier was already
+trained on some data and you want to tune (or re-tune) on a new validation set.
+
+Manually setting the decision thresholding
+-------------------------------------------
+
+The previous sections discussed strategies to find an optimal decision threshold. It is
+also possible to manually set the decision threshold in
+:class`~sklearn.model_selection.TunedThresholdClassifier` by setting the parameter
+`strategy` to `"constant"` and provide the desired threshold using the parameter
+`constant_threshold`.
+
+Examples
+--------
+
+- See
+  :ref:`sphx_glr_auto_examples_model_selection_plot_tuned_threshold_classifier.py`
+  example for an example of tuning the decision threshold of a classifier.
```

Comment:
example for an example doesn't read very well. I guess it follows up from the title in the indicated file?